eclipse/codewind-che-plugin
# Use Go Dep for package management

Signed-off-by: John Collier <John.J.Collier@ibm.com>



### What does this PR do?
Fixes https://github.com/eclipse/codewind/issues/1634.

This PR fixes the build break in the sidecar, described in https://github.com/eclipse/codewind/issues/1634. As the openshift/api and openshift/client-go have broken support for Go modules (and it isn't clear when it will be fixed), I've updated the Kube deploy code in the sidecar to use Go Dep instead of Go modules.

As the Kube deploy code in cwctl already uses Go Dep (and is based on the Kube deploy code in the sidecar), this PR uses its Gopkg.toml and Gopkg.lock (with the dependencies we didn't need removed from it).

### Link to the [Codewind repository](https://github.com/eclipse/codewind/issues) issue(s) this PR fixes or references:
https://github.com/eclipse/codewind/issues/1634

### Does this PR require updates to the docs?
Add a matching PR to [the docs repo](https://github.com/eclipse/codewind-docs) for doc updates and link that PR to this issue.
N/A

### How can this PR be tested?
Create a Che workspace with [this devfile](https://raw.githubusercontent.com/johnmcollier/devfiles/master/codewind/codewind_copy.yaml) and verify the Che workspace starts successfully.
